OvaryModelGenerator/Main.cxx

```diff
@@ -46,8 +60,37 @@ int main(int argc, char* argv[])
     
   delete pvsGen;
 
-  OvaryModelGenerator myOvary(nslices, nrot, d, h, w, outdirstr);
+  OvaryModelGenerator myOvary(nslices, nrot, d, h, w);
```

Comment:
A lot of changes in this file related to the changes in the OvaryModelGenerator source and header files. Go to those files for specific comments.